rsfmodel/rsf.py

```diff
@@ -191,6 +191,10 @@ def solve(self, **kwargs):
         self.results.displacement = np.cumsum(self.loadpoint_velocity[:-1] * dt)
         self.results.displacement = np.insert(self.results.displacement, 0, 0)
 
+        # Calculate the slider displacement
```

Comment:
FYI, if you wanted to depend on scipy (completely up to you), this process is available as `scipy.integrate.cumtrapz`.

Either way you may consider factoring out the `cumsum()` and `insert()` process into a helper function, since you have it duped here.